sdk/communication/Azure.Communication.Identity/README.md

```diff
@@ -114,10 +114,10 @@ Response revokeResponse = await client.RevokeTokensAsync(user);
 Response deleteResponse = await client.DeleteUserAsync(user);
 ```
 
-### Exchanging AAD access token of a Teams User for a Communication Identity access token
+### Exchanging Azure AD access token of a Teams User for a Communication Identity access token
```

Comment:
Similarly to the Sample1_CommunicationIdentityClient.md, I'd recommend adding a better description of how to utilize the method and how to initialize the options bag.